src/Lean/Parser/Term.lean

```diff
@@ -448,6 +448,9 @@ def withAnonymousAntiquot := leading_parser
 @[builtin_term_parser] def «trailing_parser» := leading_parser:leadPrec
   "trailing_parser" >> optExprPrecedence >> optExprPrecedence >> ppSpace >> termParser
 
+/-- Marks a function argument or return value as borrowed.
```

Comment:
This should mention that the borrowing annotation only has an effect for functions with `@[extern]`. It is ignored on all other functions.

Reply:
Done. Also fixed a docstring that made an incorrect claim about `export`.